packages/quick_actions/quick_actions/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.0+8
```

Comment:
`## NEXT`

Reply:
Shouldn't this require a version bump because of the changes I made [here](https://github.com/flutter/plugins/pull/4502/files#r750511289)?

Reply:
Yes, sorry; I confused this with a deprecated plugin.